kevm-pyk/src/kevm_pyk/gst_to_kore.py

```diff
@@ -28,6 +28,44 @@
 SORT_MODE: Final = SortApp('SortMode')
 SORT_ETHEREUM_SIMULATION: Final = SortApp('SortEthereumSimulation')
 
+GST_DISCARD_KEYS: Final = [
```

Comment:
Consider making this a `set`.

Reply:
Is there a reason to use `set` instead of `frozenset`?

Reply:
No, `frozenset` is probably even better.

Also, if these constants are not supposed to be public, then mark them with a `_`.